lib/pkcs11/portable/espressif/esp32_devkitc_esp_wrover_kit/aws_pkcs11_pal.c

```diff
@@ -51,73 +53,60 @@ enum eObjectHandles
     eAwsDeviceCertificate,
     eAwsCodeSigningKey
 };
-
 /*-----------------------------------------------------------*/
 
-static void initialize_spiffs()
+static void initialize_nvs_partition()
 {
-    static bool spiffs_inited = false;
-
-    ESP_LOGI( TAG, "Initializing SPIFFS" );
+    static bool nvs_inited;
 
     portENTER_CRITICAL();
-    if ( spiffs_inited == true )
-    {
+    if (nvs_inited == true) {
         portEXIT_CRITICAL();
         return;
     }
 
-    esp_vfs_spiffs_conf_t conf =
-    {
-        .base_path = "/spiffs",
-        .partition_label = pkcsFLASH_PARTITION,
-        .max_files = 5,
-        .format_if_mount_failed = true
-    };
+    ESP_LOGI(TAG, "Initializing NVS partition: \"%s\"", NVS_PART_NAME);
 
-    /* Use settings defined above to initialize and mount SPIFFS filesystem. */
-    /* Note: esp_vfs_spiffs_register is an all-in-one convenience function. */
-    esp_err_t ret = esp_vfs_spiffs_register( &conf );
 
-    if ( ret == ESP_OK )
-    {
-        spiffs_inited = true;
-    }
-    portEXIT_CRITICAL();
 
-    if( ret != ESP_OK )
-    {
-        if( ret == ESP_FAIL )
-        {
-            ESP_LOGE( TAG, "Failed to mount or format filesystem" );
+#if CONFIG_NVS_ENCRYPTION
```

Comment:
If someone were to enable CONFIG_NVS_ENCYRYPTION, would it work?  Or is this still a feature under test?

Reply:
Yes, it will work. This is tested feature.